packages/pigeon/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: pigeon
 description: Code generator tool to make communication between Flutter and the host platform type-safe and easier.
 repository: https://github.com/flutter/packages/tree/main/packages/pigeon
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Apigeon
-version: 7.1.4 # This must match the version in lib/generator_tools.dart
+version: 7.1.5 # This must match the version in lib/generator_tools.dart
```

Comment:
Per this comment: Update lib/generator_tools.dart as well?

Reply:
Oops, I wasn't in Pigeon mental mode; fixed.